tests/web/test_info.py

```diff
@@ -0,0 +1,5 @@
+def test_api_endpoint_shows_basic_info(client):
+    from bigchaindb import version
```

Comment:
Shouldn't it be OK to import it in the test in this case though, as you're actually using it to do an assertion?